solutions/standard/variables.tf

```diff
@@ -0,0 +1,125 @@
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The name of a new or an existing resource group to provision the watsonx.data in. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+}
+
+variable "prefix" {
+  type        = string
+  description = "(Optional) Prefix to add to all resources created by this solution. To not use any prefix value, you can set this value to `null` or an empty string."
+  default     = "data-dev"
+}
+
+variable "name" {
+  type        = string
+  description = "The name of the watsonx.data instance. If a prefix input variable is specified, the prefix is added to the name in the `<prefix>-<name>` format."
+  default     = "dev"
+}
+
+variable "region" {
+  description = "The region where you want to deploy your instance."
+  type        = string
+  default     = "us-south"
+  validation {
+    condition     = local.kms_region == null || (local.kms_region == var.region)
+    error_message = "In case of enterprise plan, KMS instance should be in the same region as watsonx.data"
+  }
+}
+
+variable "plan" {
+  type        = string
+  description = "The plan that is required to provision the watsonx.data instance. Possible values are: essentials, standard. [Learn more](https://www.ibm.com/products/watsonx-data/pricing)."
+  default     = "lakehouse-enterprise"
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to describe the newly created watsonx.data instance."
+  default     = []
+}
+
+variable "access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the watsonx.data instance. [Learn more](https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial)."
+  default     = []
+}
+
+variable "enable_kms_encryption" {
+  description = "Flag to enable KMS encryption. If set to true, a value must be passed for `existing_kms_key_crn`. This is applicable only for Enterprise plan."
+  type        = bool
+  default     = true
+
+  validation {
+    condition     = (var.enable_kms_encryption && var.existing_kms_key_crn == null) ? (var.existing_kms_instance_crn == null ? false : true) : true
+    error_message = "A value must be passed for either 'existing_kms_instance_crn' or 'existing_kms_key_crn' when 'enable_kms_encryption' is set to true."
+  }
+}
+
+
+variable "existing_kms_instance_crn" {
+  type        = string
+  default     = null
+  description = "The CRN of the existing key management service (KMS) that is used to create keys for encrypting the watsonx.data enterprise instance. If you are not using an existing KMS root key, you must specify this CRN. If you are using an existing KMS root key, an existing watsonx.data instance and auth policy is not set for watsonx.data to KMS, you must specify this CRN."
```

Comment:
`If you are using an existing KMS root key, an existing watsonx.data instance and auth policy is not set for watsonx.data to KMS, you must specify this CRN.`
- not quiet correct I think. If an user has an existing watsonx.data - then why will one use this DA?
- I guess...there is no support for existing watsonx.data